api/src/controllers/resourceController.js

```diff
@@ -34,6 +34,39 @@ const getResources = (req, res) => {
     .catch((err) => res.json({ err }).status(400))
 }
 
+// @desc    Fetch all resources
+// @route   GET /api/resources/:category?pageNumber=${pageNumber}
+// @access  Public
+const getResourcesByCategory = (req, res) => {
```

Comment:
this function is not needed only passing additional condition with req.category should work with getResources function

Reply:
Done